docs/experiment/guides/troubleshooting/variant-jumping.md

```diff
@@ -0,0 +1,92 @@
+
+The following examples are by no means exhaustive, but should hopefully get you thinking about the use of identity in your system with respect to Amplitude Experiment.
+
+### Multiple logged in accounts on a single device
+
+Consider this timeline for a user with multiple accounts for your app on a single device.
+
+1. Open app with user U1 and fetch variants, assigned `treatment` for `experiment-1`.
+2. Exposure to `experiment-1` variant `treatment` for U1.
+3. Log out of U1, and into U2, fetching variants asynchronously on log in.
+4. Prior fetch for U2 resolving, exposure to `experiment-1` variant `treatment` for U2.
+5. Fetch for U2 resolves, assigned `control` for `experiment-1`.
+6. Exposure to `experiment-1` variant `control` for U2.
+
+In this case, user U2 has jumped variants, from `treatment` to `control` due to being exposed to U1's stored variant. To avoid this sort of issue, you can either wait for the fetch to resolve before rendering the user experience, or call the SDK's `clear()` method on log out to clear all stored variants from the SDK. Clearing variants wipes the SDK's variant storage, and is a way to ensure that the user is not exposed to cached variants, but does not protect the user from viewing a fallback experience before the fetch request resolves.
+
+If you keep a consistent device ID across logins, you can easily check for this type of variant jumping by searching for different users that have the same device ID.
+
+### Inconsistent identity input between assignment and exposure
+
+In Amplitude, the user ID and device ID properties are essential to identify your user and [resolve their Amplitude ID](https://help.amplitude.com/hc/en-us/articles/115003135607-Track-unique-users-in-Amplitude#h_e9913ce1-549a-4d88-a832-1f56aede581a). If the device ID or user ID used to fetch/evaluate assignments is different from the device ID and user ID used to track the exposure event, you may see variant jumping, SRMs and inconsistent or unexpected bucketing behavior.
+
+For example, you may be sending events through a proxy or CDP which obfuscates or canonicalizes IDs before sending to Amplitude. In this case, the identity used to fetch variants would be different from the identity included in the exposure events.
+
+Another common case is simple overlooked implementation error. For example, we've seen the following cases cause variant jumping:
+
+* Additional characters in the ID. For example Extra quotes around the actual identity.
+    - `15a4f7e9-db4e-4c57-82c7-e57a2995803a`
+    - `"15a4f7e9-db4e-4c57-82c7-e57a2995803a"`
+* Inconsistent capitalization, especially with UUIDs.
+    - `15a4f7e9-db4e-4c57-82c7-e57a2995803a`
+    - `15A4F7E9-DB4E-4C57-82C7-E57A2995803A`
+
```

Comment:
Do we want to explain how to make a cohort of the users who variaint jumped and exclude then from experiment analysis? This was a possible feature we were going to add in the future.

Reply:
That's very interesting, not sure how to even do that!

Reply:
I can write up the instructions tomorrow and add it to this PR